test/__tests__/bingads_audience-cdk.test.ts

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import path from 'path';
+import {
+  processCdkV2Workflow,
+  executeStep,
+  getCachedWorkflowEngine,
+  getEmptyExecutionBindings,
+} from '../../src/cdk/v2/handler';
+import tags from '../../src/v0/util/tags';
+import { data as stepsTestData } from './data/bingads_audience_steps';
+
+const integration = 'bingads_audience';
+const destName = 'BingAds Audience';
+
+// Processor Test files
+const processorTestDataFile = fs.readFileSync(
```

Comment:
[nitpick] To import json file into our test, we can now do `import processorTestDataFile from 'my/file/path/file.json`. Rather than `fs.readFileSync`

Reply:
got it, lets do that as separate pr as that pattern we need to do in many test files.